cinema/management/commands/wait_for_db.py

```
@@ -0,0 +1,26 @@
+import time
+
+from django.core.management.base import BaseCommand
+from django.db import connections
+from django.db.utils import OperationalError
+
+
+class Command(BaseCommand):
+    """Wait command to check if a database is ready."""
+
+    def handle(self, *args, **options):
+        self.stdout.write("Waiting for database...")
+        is_connected = False
+        delay = 1
+
+        while not is_connected:
+            try:
+                connections["default"].ensure_connection()
+                is_connected = True
+            except OperationalError:
```

Catching `OperationalError` is appropriate for many cases, but in some environments other DB-related exceptions (e.g., `InterfaceError` or broader `DatabaseError`) may be raised while the DB is starting. If you observe other errors during startup, consider catching a broader exception class or logging the exception to aid debugging. Ensure you still avoid swallowing unrelated exceptions silently.